audit-opentelemetry/chart/values.yaml

```diff
@@ -0,0 +1,89 @@
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+# SPDX-License-Identifier: Apache-2.0
+
+## common labels to apply to all resources
+commonLabels: {}
+
+# opentelemetry-operator is an upstream dependency, it is required that it is a separate block. This is customised configuration of the dependency.
+opentelemetry-operator:
+  nameOverride: "operator"
+
+  # The required CRDs used by this dependency are version-controlled in this repository under ./crds.
+  # This is required because there is no other Plugin that would install them, and because automatic CRD installs are not supported due to a limitation with helm, see [HIP0011](https://github.com/helm/community/blob/main/hips/hip-0011.md).
+  crds:
+    create: false
+  admissionWebhooks:
+    failurePolicy: 'Ignore'
+
+    # If cert-manager should be used, the CRDs for cert-manager needs to exists before otel installs cert-manager.
+    certManager:
+      enabled: false
+    autoGenerateCert:
+      recreate: false
+  manager:
+    collectorImage:
+      repository: ghcr.io/cloudoperators/opentelemetry-collector-contrib
+      tag: main
+    image:
+      repository: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
+      tag: v0.114.1
+    deploymentAnnotations:
+      vpa-butler.cloud.sap/update-mode: Auto
+    serviceMonitor:
+      enabled: true
+  kubeRBACProxy:
+    enabled: false
+
+# Empty variables will fail during rollout
+openTelemetry:
```

Comment:
I would suggest here to change the name even to audit-openTelemetry (or similar) so we can dinstinguish it a bit better